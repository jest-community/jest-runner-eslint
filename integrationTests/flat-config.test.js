const { version } = require('eslint/package.json');
const semver = require('semver');
const runJest = require('./runJest');

(semver.satisfies(version, '>=8.41') ? it : it.skip)(
  'Works with the new flat config format',
  async () => {
    expect(await runJest('flat-config')).toMatchSnapshot();
  },
);
