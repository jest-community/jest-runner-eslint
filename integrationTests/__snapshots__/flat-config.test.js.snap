// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works with the new flat config format 1`] = `
"FAIL __eslint__/file.js
  ✕ no-console 


/mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/flat-config/__eslint__/file.js
  3:1  error  Unexpected console statement  no-console

✖ 1 problem (1 error, 0 warnings)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;
